pkg/kubelet/server/stats/volume_stat_calculator.go

```diff
@@ -142,9 +143,14 @@ func (s *volumeStatCalculator) calcAndStoreStats() {
 		if err != nil {
 			// Expected for Volumes that don't support Metrics
 			if !volume.IsNotSupported(err) {
-				klog.V(4).InfoS("Failed to calculate volume metrics", "pod", klog.KObj(s.pod), "podUID", s.pod.UID, "volumeName", name, "err", err)
+				if oldErr, found := errors[name]; !found || oldErr.Error() != err.Error() {
```

Comment:
there should be else statement that will log the message with high verbosity so if we need to debug it we can see every instance of the error